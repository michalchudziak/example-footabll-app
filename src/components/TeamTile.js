/* @flow */
import { Dimensions } from 'react-native';
import styled from 'styled-components/native';

const TeamTile = styled.Text`
  padding-vertical: 20;
  padding-left: 10;
  width: ${Dimensions.get('window').width};
  background-color: #E6E6FA;
  font-size: 16;
`;

export default TeamTile;
