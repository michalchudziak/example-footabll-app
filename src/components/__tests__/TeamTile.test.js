/* @flow */
import React from 'react';
import renderer from 'react-test-renderer';
import TeamTile from '../TeamTile';

describe('<TeamTile />', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<TeamTile>{'Test'}</TeamTile>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
