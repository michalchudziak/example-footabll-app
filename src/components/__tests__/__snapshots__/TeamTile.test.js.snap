// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TeamTile /> should render correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "backgroundColor": "#E6E6FA",
        "fontSize": 16,
        "paddingLeft": 10,
        "paddingVertical": 20,
        "width": 750,
      },
      undefined,
    ]
  }
>
  Test
</Text>
`;
